code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -27,12 +27,11 @@
 		var/list/meancritters = list() // list of possible hostile mobs
 		var/list/nicecritters = list() // and possible friendly mobs
 		for(var/T in typesof(/mob/living/simple_animal))

```

Comment:
As far as I know, that won't work, Or at least, I remember being told that you can't avoid the color operator in this case